zanata-model/src/main/java/org/zanata/model/HProject.java

```diff
@@ -143,14 +156,30 @@
     private ProjectType defaultProjectType;
 
     /**
+     * Immutable list of maintainers for this project.
```

Comment:
It returns a set not a list. May also change method signature to return ImmutableSet instead so people don't need to look at javadoc to figure out it's immutable.